apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/wpcom-global-styles-view.js

```diff
@@ -23,13 +24,25 @@ document.addEventListener( 'DOMContentLoaded', () => {
 	}
 	const popoverToggle = container.querySelector( '.launch-bar-global-styles-toggle' );
 	const popover = container.querySelector( '.launch-bar-global-styles-popover' );
-	const upgradeButton = container.querySelector( '.launch-bar-global-styles-upgrade-button' );
-	const previewButton = container.querySelector( '.launch-bar-global-styles-preview-link' );
+	const upgradeButton = container.querySelector( '.launch-bar-global-styles-upgrade' );
+	const previewButton = container.querySelector( '.launch-bar-global-styles-preview' );
+
+	const showLimitedGlobalStylesNotice =
+		localStorage.getItem( 'showLimitedGlobalStylesNotice' ) === null;
+	if ( showLimitedGlobalStylesNotice ) {
+		popover?.classList.remove( 'hidden' );
+		recordEvent( 'wpcom_global_styles_gating_notice', {
+			action: 'show',
+		} );
+	}
 
 	popoverToggle?.addEventListener( 'click', ( event ) => {
 		event.preventDefault();
-		recordEvent( 'wpcom_global_styles_gating_notice' );
+		recordEvent( 'wpcom_global_styles_gating_notice', {
+			action: popover?.classList.contains( 'hidden' ) ? 'show' : 'hide',
+		} );
 		popover?.classList.toggle( 'hidden' );
+		localStorage.setItem( 'showLimitedGlobalStylesNotice', '0' );
```

Comment:
Maybe the value should be toggled instead of always being persisted as '0'? If the user leaves it open, then it should remain opened? 🤔

Reply:
https://github.com/Automattic/wp-calypso/pull/76459/commits/88331cd28674f165f5402fe409622f2d38de87e1